cpp/ql/src/Critical/UseAfterFree.ql

```diff
@@ -31,7 +31,7 @@ private predicate externalCallNeverDereferences(FormattingFunctionCall call, int
 
 predicate isUse0(DataFlow::Node n, Expr e) {
   e = n.asExpr() and
-  not isFree(_, e, _) and
+  not isFree(n, _, _) and
```

Comment:
This makes me a bit nervous. because `isFree` looks a (un)converted expressions explicitly. Was the change just made to bind `n`, or is there a deeper reason? If the former, can't we just drop the node argument of `isUse0` completely?

Reply:
I think the change was leftover from an earlier attempt which I accidentality left in. I have now changed it to not have a `DataFlow::Node` argumebnt at all and call `asExpr` at call sites.